recipes/librhash/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+import os
+from conans import CMake, ConanFile, tools
+
+
+class LibRHashConan(ConanFile):
+    name = "librhash"
+    description = "Great utility for computing hash sums"
+    topics = ("conan", "rhash", "hash", "checksum")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://rhash.sourceforge.net/"
+    license = "MIT"
+    exports_sources = "CMakeLists.txt",
+    generators = "cmake",
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_openssl": [True, False]}
+    default_options = {"shared": False, "fPIC": True, "with_openssl": False,}
+
+    _source_subfolder = "source_subfolder"
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+        if self.settings.os == "Windows" or self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.options.with_openssl:
+            self.requires("openssl/1.1.1d")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = "RHash-{}".format(self.version)
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def _get_cmake(self):
```

Comment:
name this as usual?
```suggestion
    def _configure_cmake(self):
```

Reply:
Same argument as https://github.com/conan-io/conan-center-index/pull/281#discussion_r342097697

Reply:
I renamed to `_configure_cmake`